app/controllers/internal/profile_fields_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Internal
+  class ProfileFieldsController < Internal::ApplicationController
+    layout "internal"
+
+    def index
+      @profile_fields = ProfileField.all
+    end
+
+    def update
+      @profile_fields = ProfileField.find(params[:id])
```

Comment:
**nitpick (non-blocking):** Since this is the update action, this should probably be singular, i.e. `profile_field`. It probably doesn't need to be an instance variable either, as we're not rendering in `update`.